nevergrad/benchmark/test_experiments.py

```diff
@@ -26,15 +28,36 @@
 
 @testing.parametrized(**{name: (name, maker) for name, maker in experiments.registry.items()})
 def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[experiments.Experiment]]) -> None:
+    # Our PGAN is not well accepted by circleci.
+    if "_pgan" in name and os.environ.get("CIRCLECI", False):
+        raise SkipTest("Too slow in CircleCI")
+
+    # Our IQAs and our ScikitLearn are not well guaranteed on Windows.
+    if "image" in name and "quality" in name and platform.system() == "Windows":
+        raise SkipTest("Image quality not guaranteed on Windows.")
+    if "tuning" in name and platform.system() == "Windows":
+        raise SkipTest("Image quality not guaranteed on Windows.")
+
+    # Basic test.
     with tools.set_env(NEVERGRAD_PYTEST=1):
         with datasets.mocked_data():  # mock mlda data that should be downloaded
             check_maker(maker)  # this is to extract the function for reuse if other external packages need it
-    if name not in ["rocket", "images_using_gan", "control_problem"] and not any(
-        x in name for x in ["tuning", "mlda", "realworld", "image_"]
+
```

Comment:
we were skipping rocket, images_* image_*, realworld, mlda, tuning, control_problem.

I still skip control_problem.
For others, I try to skip minimally: remove 
- the seed testing for rocket and images_using_gan;
- image quality on Windows;
- gan, mlda, realworld, only on the CI.